projects/kit/styles/components/message.less

```diff
@@ -0,0 +1,38 @@
+@import '@taiga-ui/core/styles/taiga-ui-local.less';
+
+/**
+ * @name Message
+ * @selector [tuiMessage]
+ *
+ * @description
+ * A message component that can be used on buttons/links/inputs
+ *
+ * @example
+ * <span tuiMessage>Message text</span>
+ *
+ * @see-also
+ * Chip, Badge, Block
+ */
+[tuiMessage] {
+    display: inline-flex;
+    padding: 0.5rem 0.625rem;
+    min-block-size: 2.25rem;
+    block-size: auto;
+    box-sizing: border-box;
+    inline-size: fit-content;
+    isolation: isolate;
+    white-space: nowrap;
+    text-align: start;
+    font: var(--tui-font-text-ui-m);
+    border-radius: var(--tui-radius-l);
+
+    :host-context(tui-root._mobile) & {
```

Comment:
Should we maybe introduce mobile specific design using tuiPlatform and inside mobile theme styles?

Reply:
why tuiPlatofrm? to add example to demo?